overview/3-creating-packages.md

```diff
@@ -1,47 +1,10 @@
 ---
-title: Creating & Saving Packages
+title: Creating Packages
 layout: default
 permalink: "creating-packages.html"
 parent: Overview
 ---
 
-This sections presents an overview of package management with Helios.
-
 <p class="warning">
-This page is a work in progress.
+This page is waiting for you to write it.
 </p>
-
```

Comment:
I think this page got deleted for some reason.
@hhzl do you want to pull, merge and push so we can be sure?